orangecanvas/canvas/items/nodeitem.py

```diff
@@ -4,14 +4,16 @@
 =========
 
 """
+import math
 import typing
 import string
+import numpy as np
```

Comment:
`numpy` is currently not a requirement of `orange-canvas-core`.
Either we add the dependency or the code needs to be changed to not use numpy.

Reply:
I'll eliminate the numpy and scipy dependencies.